tests/src/FunctionalJavascript/JavascriptBehavioursTest.php

```diff
@@ -123,7 +123,7 @@ public function testEclMultiSelect(): void {
 
     // Assert all options are visible.
     $options = [
-      'Select all',
+      'Select all (4)',
```

Comment:
Why is this happening? We cannot just change the test cause drone fails. There is a reason we have tests and that is to check if we break stuff. Not that if we break stuff, we just adapt the test. So please confirm what this is.